.circleci/config.yml

```diff
@@ -488,55 +497,134 @@ commands:
             - run:
                 name: Wait for memcached
                 command: dockerize -wait tcp://localhost:11211 -timeout 1m
+      - run: pip install --no-deps -r requirements/ci_base.txt
```

Comment:
With the removal of `tox`, we shouldn't need `ci_base.txt` anymore. Running `make install_python_test_dependencies` should install `system.txt` and `tests.txt` which should contain everything we need.

Reply:
So, we should move `toml` to `system.txt` I guess?

Reply:
Do we still need `wheels` and `setuptools`?

Reply:
I would move `wheel` to `system.txt`.

Reply:
and `toml` is already in `tests.txt` so it should be ok.

Reply:
> I would move `wheel` to `system.txt`.

looks like it's not even needed?

Reply:
Looks like you're right!